apps/andi/test/support/dataset_helpers.ex

```diff
@@ -6,7 +6,7 @@ defmodule DatasetHelpers do
   alias Andi.InputSchemas.Datasets
   alias Andi.InputSchemas.Datasets.Dataset
 
-  require Placebo
+  import Placebo
```

Comment:
wooooow

Reply:
Yea this is caused by something that changed in placebo which makes require not work because inside its macro it calls some of its own functions as local members.